Files/DataModels/SidebarPinnedModel.cs

```diff
@@ -70,7 +70,7 @@ public void AddDefaultItems()
         {
             FavoriteItems.Add(AppSettings.DesktopPath);
             FavoriteItems.Add(AppSettings.DownloadsPath);
-            FavoriteItems.Add(AppSettings.DocumentsPath);
+            // FavoriteItems.Add(AppSettings.DocumentsPath); // TODO: add library path?
```

Comment:
These properties are removed from AppSettings as they give us the default save path of the initially existing libraries only. This line could be kept here (if the propery is re-added), but it won't be recognized as the Documents library because of other changes. So it would be better to add the library path of Documents here if needed, I can make a commit for it.

Reply:
Let's do it properly, can you add a commit for this?

Reply:
@nvi9 I have a couple other changes that I want to make, do you mind if I push a couple commits?

Reply:
Now you can, I've also pushed some fixes.

I've tried to do the Documents library pinning, but it would require more changes than expected, because `SidebarPinnedController` is initialized before the libraries are loaded here:

https://github.com/files-community/Files/blob/8f7c27bb3f2a5480d280e0ecb2a8de7cf3740a80/Files/App.xaml.cs#L98-L111